src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNICommon.cs

```diff
@@ -343,6 +343,7 @@ internal static IPAddress[] GetDnsIpAddresses(string serverName, TimeoutTimer ti
                                                           args1: remainingTimeout);
                 using CancellationTokenSource cts = new CancellationTokenSource(remainingTimeout);
                 // using this overload to support netstandard
+                //TODO Refactor?
```

Comment:
@DavoudEshtehari Should this now be changed to: ?

return Dns.GetHostAddressesAsync(serverName, cts.Token)
    .ConfigureAwait(false).GetAwaiter().GetResult();

Reply:
My apologies for the late response. For sure, it should be.